addons/delivery/models/delivery_carrier.py

```diff
@@ -183,6 +184,13 @@ def log_xml(self, xml_string, func):
             except psycopg2.Error:
                 pass
 
+    def _get_default_custom_package_code(self):
```

Comment:
""" Some delivery carriers require a prefix to be sent in order to use custom
packages (ie not official ones). This optional method will return it as a string.
"""